devtools_m2/setup.sh

```diff
@@ -31,7 +31,30 @@ MAGE_MODE=developer ./bin/magento setup:install \
 SCRIPT
 )
 
-docker-compose exec -T -u www-data web /bin/bash -c "$magento_setup_script"
+./docker_compose.sh exec -T -u www-data web /bin/bash -c "$magento_setup_script"
 
-# Backup for reset.
-docker-compose exec -e MYSQL_PWD=magento db mysqldump -u magento magento > db_data/dump.sql
+# For multi-store.
+./docker_compose.sh exec -T -u www-data web patch -p1 /var/www/html/magento/index.php <<'SCRIPT'
```

Comment:
Doing a patch seems brittle... but maybe it's good for it to break quickly.

Reply:
That was my thought. This change is the official way to enable multi-site (although I might be able to do something with environment variables in some cases). The patch will break immediately if they change index.php.